tools/code_format/check_format.py

```diff
@@ -7,17 +7,21 @@
 import os
 import pathlib
 import re
-import subprocess
+import shutil
 import stat
+import subprocess
 import sys
 import traceback
-import shutil
-from functools import cached_property
-from typing import Callable, Dict, List, Pattern, Tuple, Union
-
 # The way this script is currently used (ie no bazel) it relies on system deps.
 # As `pyyaml` is present in `envoy-build-ubuntu` it should be safe to use here.
 import yaml
+from functools import cached_property
+from typing import Callable
+from typing import Dict
+from typing import List
+from typing import Pattern
+from typing import Tuple
+from typing import Union
```

Comment:
The previous version of these imports seems more concise. Did format fix this or, is this a design choice.

Reply:
It was format fixed. Happy to revert if previous was closer to Enovy's style

Reply:
reverted